packages/plugin-multi-tenant/src/translations/languages/de.ts

```diff
@@ -5,6 +5,7 @@ export const deTranslations: PluginDefaultTranslationsObject = {
     'confirm-tenant-switch--body':
       'Sie sind dabei, den Besitz von <0>{{fromTenant}}</0> auf <0>{{toTenant}}</0> zu übertragen.',
     'confirm-tenant-switch--heading': 'Bestätigen Sie die Änderung von {{tenantLabel}}.',
+    'field-assignedTentant-label': 'Zugewiesener Mieter',
```

Comment:
As a native German speaker, I’d recommend using „Mandant“ instead of „Mieter“. While "Mieter" translates to "renter" and is commonly associated with renting an apartment, „Mandant“ is the established term in IT for a tenant in a multi-tenant system. See [Wikipedia – Mandantenfähigkeit](https://de.wikipedia.org/wiki/Mandantenf%C3%A4higkeit) for reference.

Reply:
Thanks!